hypergan/cli.py

```diff
@@ -295,7 +295,7 @@ def run(self):
         config['x_dims']=[height,width]
         config['channels']=channels
 
-        if(int(config['y_dims']) > 1):
+        if(int(config['y_dims']) > 1 and not args.align):
```

Comment:
Do we want to support alignment while also building a classifier?  I don't see why not.